eureka-client/src/main/java/com/netflix/discovery/converters/EurekaJacksonCodec.java

```diff
@@ -230,7 +230,7 @@ public Name getName() {
                 };
             }
 
-            Map<String, String> metaData = new HashMap<>();
+            Map<String, String> metaData = new HashMap<String, String>();
```

Comment:
Actually most of Eureka (1.0) is at Java6, and since it has some pretty old users it would be nice to keep it that way.